setup.py

```diff
@@ -3,8 +3,9 @@
 
 
 install_requires = [
-    "fastplotlib[notebook]==0.1.0a16",
-    "pygfx==0.1.17",
+    "fastplotlib[notebook]",
+    "pygfx",
+    "imgui_bundle",
```

Comment:
can just specify `fastplotlib[notebook,imgui]`: https://github.com/fastplotlib/fastplotlib?tab=readme-ov-file#installation

don't need to specify pygfx since we pin that when neccessary in fastplotlib: https://github.com/fastplotlib/fastplotlib/blob/main/setup.py#L7

just having fastplotlib will get the right version of pygfx.

Reply:
[ca3846a](https://github.com/kushalkolar/mesmerize-viz/pull/45/commits/ca3846ac612d94bd607a029607fd41df1ea60a01)